models/field_definition.go

```diff
@@ -8,20 +8,20 @@ import (
 
 // constants for describing possible field types
 const (
-	KindString            Kind = 1
-	KindInteger           Kind = 2
-	KindFloat             Kind = 3
-	KindInstant           Kind = 4
-	KindDuration          Kind = 5
-	KindURL               Kind = 6
-	KindWorkitemReference Kind = 7
-	KindUser              Kind = 8
-	KindEnum              Kind = 9
-	KindList              Kind = 10
+	KindString            Kind = "string"
+	KindInteger           Kind = "integer"
+	KindFloat             Kind = "float"
+	KindInstant           Kind = "instant"
+	KindDuration          Kind = "duration"
+	KindURL               Kind = "url"
+	KindWorkitemReference Kind = "work_item_reference"
```

Comment:
Not sure underscore is the most userfriendly here. Nor do we know of this is really a Kind at all.. :)

But I think "workitem" is better for now.